### PR TITLE
inital checkin

### PR DESCRIPTION
**Pull request by bemeyert**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,6 +4,8 @@ CI_REGISTRY ?= docker.io
 IMAGE_PREFIX ?= $(CI_REGISTRY)/7val/
 CURRENT_BRANCH ?= $(shell git symbolic-ref --short HEAD)
 ONLY_BRANCH ?= master
+# needed to mount files inside a container started inside a container
+HOST_PATH=$(shell pwd)
```
Comment (kr1sp1n) [inline Makefile]: When using `.EXPORT_ALL_VARIABLES:` before the variables definitions then you could leave out the `@export HOST_PATH=$(HOST_PATH);` in so many places.

File: sloppy-rollout/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# 7val/sloppy-deployment
```
Comment (kr1sp1n) [inline sloppy-rollout/README.md]: Should be `7val/sloppy-rollout`.

**Comment by bemeyert** [comment]
@kr1sp1n I added the required refinements. Please check again.

File: README.md [hunk 3]
```diff
@@ -25,7 +25,10 @@ Opinionated conventions and best practices:
 * [7val/rancher-deployment]: Run a deployment to [Rancher][1] via generated
   `docker-compose.yml` and `rancher-compose.yml` files.
 * [7val/sloppy]: Provides the Slopp.io CLI in a container
-* [7val/sloppy-deployment]: Run a Sloppi.io deployment
+* [7val/sloppy-deployment]: Run a Sloppy.io deployment with the help of template
+  configuration and environment variables.
+* [7val/sloppy-rollout]: Enhances `7val/sloppy-deployment`. Plain JSON/YAML
```
Comment (kr1sp1n) [inline README.md]: You need to add the definition of the reference `[7val/sloppy-rollout]` at the end of the file. Otherwise it is not a clickable link.

Comment (bemeyert) [reply README.md]: fixed

**Comment by bemeyert** [comment]
@kr1sp1n Added the missing link. Anything else? ;)

**Pull request merged**